keda/Chart.yaml

```diff
@@ -8,11 +8,11 @@ kubeVersion: ">=v1.20.0-0"
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.8.1
+version: 2.8.2
```

Comment:
Please revert this as per https://github.com/kedacore/keda/issues/3626#issuecomment-1234109552

https://github.com/kedacore/charts/releases/tag/v2.8.1

Reply:
Wasn't your bump already released? https://github.com/kedacore/charts/commit/b0eb796ae604fda2b6492f3a693dd8382bd6955e#diff-22efdd45972c44fc1f4743550c58a1d82b372e7c2ba340c5ae89d832538aacbf

Reply:
You are correct, I thought this PR was already merged!
https://github.com/kedacore/charts/pull/312/files#diff-22efdd45972c44fc1f4743550c58a1d82b372e7c2ba340c5ae89d832538aacbfR11

We can merge it and change the readme before shipping unless you prefer to move on?

Reply:
I'll do it